Makefile

```diff
@@ -3,7 +3,7 @@ CXXFLAGS=-Wall -Werror
 CP_LOC=config_parser/
 
 all: server.o session.o main.o config_parser.o
-	g++ -o web-server main.o server.o session.o config_parser.o $(LDFLAGS) $(CXXFLAGS)
```

Comment:
small thing: you think we should add -std=c++0x to the $(CXXFLAGS) variable since its appearing in multiple places?
I'd change it on mine but might as well just use the pull request